src/compass/utils/geo_metadata.py

```diff
@@ -265,3 +265,75 @@ def to_file(self, dst, fmt:str):
             json.dump(self_as_dict, dst, indent=4)
         else:
             raise ValueError(f'{fmt} unsupported. Only "json" or "yaml" supported')
+
+    def to_hdf5(self, dst_h5):
+        '''Write self to HDF5
+
+        Parameter:
+        ---------
+        dst_h5: h5py File
+            HDF5 file meta data will be written to
+        '''
+        metadata_group = dst_h5.require_group('metadata')
+
+        # subset of burst class attributes
+        metadata_group['sensing_start'] = np.string_(self.sensing_start.__str__)
```

Comment:
I think that to satisfy CF convention, all these metadata need to have a description and unit as attributes (see NISAR specs on NISAR-PIX)

Reply:
I think that to satisfy the CF convention we should have something of this sort: https://github.com/opera-adt/RTC/blob/93c3118ea4cb91058c4afc0a9e410444b038a57f/src/rtc/h5_prep.py#L46. May be we should have a function writing and adding metadata to the HDF5 file